src/pombot/index.js

```diff
@@ -23,21 +24,34 @@ const bot = createSlackBot({
   createMessageHandler(id, {channel}) {
     // Direct message
     if (channel.is_im) {
-      return createConversation([
-        // Nameless command that encapsulates sub-commands and adds a "help"
-        // command and a fallback message handler.
-        createCommand({
-          isParent: true,
-          description: `Hi, I'm pombot!`,
-        }, [
-          startCommand,
-          stopCommand,
-          iwillCommand,
-          statusCommand,
-        ]),
+      // create the pom for this message
+      const pom = new Pom({
+        maxMinutes: 1,
+        warningMinutes: 0.5,
+        onWarningCallback: () => {
+          this.sendResponse({channel}, `warning: there are *${pom.timeLeft / 60000}* minutes left in this pom!`);
```

Comment:
`sendResponse` expects a first argument like `{channel: 'D0JJE9TBR'}` (the channel ID string).

Try `{channel: channel.id}` instead.